src/main/java/cheifetz/scrabble/webapp/index.html

```diff
@@ -0,0 +1,17 @@
+<HTML>
```

Comment:
This file is not in the correct directory

Reply:
Where is it supposed to go? You said in class webapp